node/config/types.go

```diff
@@ -546,6 +547,9 @@ type FevmConfig struct {
 	// Set to 0 to keep all mappings
 	EthTxHashMappingLifetimeDays int
 
+	// EthTraceFilterMaxResults sets the maximum results returned per request by trace_filter
+	EthTraceFilterMaxResults ethtypes.EthUint64
```

Comment:
```suggestion
	EthTraceFilterMaxResults uint64
```

sorry I didn't catch this earlier, we probably shouldn't be bringing in custom types here, especially ones (like this one) with custom JSON marshalling. This would allow you to put `0xffff` in your config file to specify a plain integer.